tests/Subscribe.js

```diff
@@ -1,46 +1,46 @@
-import Menu from "../page-objects/components/Menu"
-import AllArticles from "../page-objects/pages/AllArticles"
-import CutePage from "../page-objects/pages/CutePage"
-import Homepage from "../page-objects/pages/Homepage"
-import SubscribePage from "../page-objects/pages/SubscribePage"
+import Menu from "../page-objects/components/Menu";
+import AllArticles from "../page-objects/pages/AllArticles";
+import CutePage from "../page-objects/pages/CutePage";
+import Homepage from "../page-objects/pages/Homepage";
+import SubscribePage from "../page-objects/pages/SubscribePage";
 
-describe('Subscribe', ()=>{
-    beforeEach(async () => {
-        
-        await Homepage.openHomePage()     
-         })
-    it('Subscribe on the articles page', async() =>{
-        await Menu.approveButton.click()
-        await Menu.cuteButton.click()
-        await browser.pause(4000)
-        await CutePage.subscribeLabel.addValue(SubscribePage.email())
-        await CutePage.emailButton.click()
-        await expect(SubscribePage.confirmSubscribe).toBeDisplayed()
-    })
+describe("Subscribe", () => {
+  beforeEach(async () => {
+    await Homepage.openHomePage();
+  });
+  it("Subscribe on the articles page", async () => {
+    await Menu.approveButton.click();
+    await Menu.cuteButton.click();
+    await CutePage.subscribeLabel.waitForDisplayed();
+    await (await CutePage.subscribeLabel).addValue(SubscribePage.email());
+    await CutePage.emailButton.click();
+    await expect(SubscribePage.confirmSubscribe).toBeDisplayed();
```

Comment:
When this steps will fail there is be no error message. As a error message will be like selector.for.confirm.Subscribe not displayed. Maybe you can add a error message for expect, e.g.  
expect(SubscribePage.confirmSubscribe, 'The subscribe for was not displayed').toBeDisplayed();
This is a comment for all the expect where you think it is needed.